### DISCUSSION TITLE
Missing function in python binding.

### DISCUSSION THREAD

**Discussion by xkiixkii** [google-deepmind/mujoco#2106] [Asking for Help]
### Intro

Hi!

I am a student tring to use MuJoCo to create a reinforcement learning environment for stacking deformable objects.

### My setup

MuJoCo Version: 3.1.6
OS: Ubuntu 22.04.4 x86_64
Using mujoco python binding

### My question

I want to spawn some soft objects on some random positions at runtime.
My intended method is to convert MjModel to Mjspec, add these soft objects, and then recompile it back to MjModel. However, I found that the exposed Python API does not provide the `mj_copyBack`function, and some other functions mentioned in the documentation are also not exposed in the Python API. Is there any other way to achieve the same effect as the `mj_copyBack` `function?"

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
> My intended method is to convert MjModel to Mjspec

This is not possible, and not what mj_copyBack does. You need to start with an mjSpec, modify that, and compile to mjModel when required.

The whole copyback functionality is a remnant from before mjSpec existed and doesn't really makes sense anymore. There is now no reason to modify mjMode. We'll probably delete it in the future.
